internal/os_util.py

```diff
@@ -128,3 +129,21 @@ def get_file_version(filename):
     except:
         logging.exception('Error getting file version for %s', filename)
     return version
+
+def util_remove_file(_file):
```

Comment:
I'd get rid of the util_ prefix and make it just remove_file

Reply:
> 

I will be following up with another PR to change these locations out. Just wanted to get an opinion on it first.